iotfunctions/anomaly.py

```diff
@@ -598,6 +622,13 @@ def build_ui(cls):
                 description='Size of each sliding window in data points. Typically set to 12.'
                                               ))
 
+        inputs.append(UIExpression(
+                name='expression',
+                description="Define alert expression using pandas systax. \
```

Comment:
The description confuses me: The example 'inlet_temperature' does not have ${}. The next sentence makes sense but why pressure? The tail is definitely wrong: ${} is never substituted with df[<dimension_name>].
I have already seen many expressions that do not use the ${} approach which is not mandatory anymore. It's good to have the feature in the code for backwards compatibility but we should not mention it anymore in the description.

Reply:
Perils of copy/paste